### PR TITLE
UI with currency exchange rate

### PR DESCRIPTION
**Pull request by Tatiana-256**
Added table to show currency

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,11 +3,14 @@
...
   "private": true,
   "dependencies": {
+    "@emotion/react": "^11.10.4",
+    "@emotion/styled": "^11.10.4",
+    "@mui/material": "^5.10.6",
     "dotenv": "^16.0.2",
     "express": "^4.18.1",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
-    "react-scripts": "5.0.1",
```
Comment (singvarr) [inline package.json]: Why did you downgrade `react-scripts`?

File: src/Table/hook.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import { useEffect, useState } from "react";
+
+export const useTable = () => {
+  const [loading, setLoading] = useState(true);
+  const [error, setError] = useState("");
+  const [currency, setCurrency] = useState([]);
+
+  useEffect(() => {
+    fetch("http://localhost:3000")
```
Comment (singvarr) [inline src/Table/hook.js]: rename env variable `SERVER_PORT to `REACT_APP_SERVER_PORT` and use here

File: src/Table/Table.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import Table from "@mui/material/Table";
+import TableBody from "@mui/material/TableBody";
+import TableCell from "@mui/material/TableCell";
+import TableContainer from "@mui/material/TableContainer";
+import TableHead from "@mui/material/TableHead";
+import TableRow from "@mui/material/TableRow";
+import Paper from "@mui/material/Paper";
+import styles from "./styles.module.css";
+import { useTable } from "./hook";
```
Comment (singvarr) [inline src/Table/Table.js]: 1. Don't move to a separate hook, put all logic in the component
2. Don't worry about error handling

File: src/Table/Table.js [hunk 4]
```diff
@@ -0,0 +1,53 @@
+import React from "react";
```
Comment (singvarr) [inline src/Table/Table.js]: There is no need to import `react` everywhere

File: src/Table/Table.js [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+  const { loading, error, currency: rows } = useTable();
+
+  if (error) {
+    return <div className={styles.info}>{error}</div>;
+  }
+
+  return (
+    <TableContainer
+      component={Paper}
+      sx={{ width: "70%" }}
```
Comment (singvarr) [inline src/Table/Table.js]: remove `sx`, move all styles to CSS-module. I want to keep code with logic clean and simple

**Review by singvarr — CHANGES_REQUESTED** [review]
Fix small codestyle issue, please

File: package.json (hunk 1 above)
Comment (Tatiana-256) [reply package.json]: changed back

File: src/Table/hook.js (hunk 2 above)
Comment (Tatiana-256) [reply src/Table/hook.js]: done

File: src/Table/Table.js (hunk 3 above)
Comment (Tatiana-256) [reply src/Table/Table.js]: done

File: src/Table/Table.js (hunk 4 above)
Comment (Tatiana-256) [reply src/Table/Table.js]: done

File: src/Table/Table.js (hunk 5 above)
Comment (Tatiana-256) [reply src/Table/Table.js]: removed

**Pull request merged**